### ISSUE TITLE
Reproduction failed.

### ISSUE THREAD

**Issue by One-T0505** [ChatAFLndss/ChatAFL#7]
I have successfully build the docker images for subjects and run this command:
 ./run.sh 5 240 kamailio,pure-ftpd,live555,lighttpd1,exim chatafl,aflnet
 
 but the results are extremely strange, because i query my openai api usage, it is 0.  Even i set the maxPlateau to 64, it is the same.

**Comment by One-T0505** [comment]
I successfully run the model and run the analysis.py script, but the results are extreamly strange, does it really use the LLM? 
Because i replace the Openai API, but with zero API calls.

**Comment by Marti2203** [comment]
Hi, sorry for the late response. Can you provide more information about the issue? One source of debug info is the logs of the created Docker containers for the fuzzers. Another way to check that everything is set up correctly is to execute the `run.sh` script manually. I have just done the latter and can see no issues with the master branch. 
To be sure, you have followed the setup instructions outlined on the page, correct?

**Comment by One-T0505** [comment]
yes, i followed the instructions in the README.  Maybe it's my server's problem, because i switch to another server in different cloud platform, it seems compile correctly.  But there are still new problems:
 1. all experiments seem do not call my openai API for use when i visit the api platform.
 2. when i execute `sudo docker logs -f <container_id>` , it logs error like:
     **(INFO) Reading coverage data...
        (INFO) Writing coverage report...
        rm: cannot remove '/home/ubuntu/ftpshare/*': No such file or directory
        2024-09-07 02:40:40,193 8677e59c1626 proftpd[159164] 8677e59c1626: mod_delay/0.8: error opening DelayTable 
            '/usr/local/var/proftpd.delay': No such file or directory
        2024-09-07 02:40:40,202 8677e59c1626 proftpd[159164] 8677e59c1626: ProFTPD 1.3.9rc1 (git) (built Fri Sep 6 2024 
             15:46:30 UTC) standalone mode STARTUP
        2024-09-07 02:40:40,229 8677e59c1626 proftpd[159164] 8677e59c1626 (localhost[127.0.0.1]): mod_delay/0.8: 
             unable to open DelayTable '/usr/local/var/proftpd.delay': No such file or directory
        2024-09-07 02:40:40,230 8677e59c1626 proftpd[159164] 8677e59c1626 (localhost[127.0.0.1]): FTP session 
            opened.**

**Comment by One-T0505** [comment]
it is extremely strange. i think i have successfully run the experiments, but the results are strange:
![cov_over_time_exim](https://github.com/user-attachments/assets/6e181e00-bf8f-4978-a7c2-119ac00e88f9)

**Comment by kr-jq** [comment]
I have the same issue and don't know if you have solved these issues now

**Comment by One-T0505** [comment]
> I have the same issue and don't know if you have solved these issues now

no，i dont know how to solve this problem，and waiting for the author‘s reply。
